docs/book/getting-started/deploying-zenml/cli.md

```diff
@@ -261,4 +261,21 @@ To disconnect from the current ZenML server and revert to using the local defaul
 
 ```bash
 zenml disconnect
-```
\ No newline at end of file
+```
+
+## Upgrading your ZenML server
+
+To upgrade your ZenML server (that was deployed with the `zenml deploy` command) to a newer version, you can follow the steps below.
+
+- In the config file, set `zenmlserver_image_tag` to the version that you want your ZenML server to be running.
+- Run the deploy command again with this config file:
+    
+    ```bash
+    zenml deploy --config=/PATH/TO/FILE
+    ```
```

Comment:
This suggestion isnt formatted properly please dont accept it from github but fix manually 
```suggestion

```bash
zenml deploy --config=/PATH/TO/FILE
```

```